src/sql/src/session/vars.rs

```diff
@@ -648,6 +648,22 @@ pub const AUTO_ROUTE_INTROSPECTION_QUERIES: ServerVar<bool> = ServerVar {
     safe: true,
 };
 
+pub const MAX_SUBSCRIBES: ServerVar<u32> = ServerVar {
+    name: UncasedStr::new("max_subscribes"),
+    value: &100,
+    description: "The maximum number of concurrent subscribes on a single cluster (Materialize).",
```

Comment:
```suggestion
    description: "The maximum number of concurrent subscribes (Materialize).",
```

Reply:
I think we need to remove "on a single cluster" because these limits are system wide.